pkg/executor/insert.go

```diff
@@ -149,6 +149,9 @@ func prefetchUniqueIndices(ctx context.Context, txn kv.Transaction, rows []toBeC
 			batchKeys = append(batchKeys, k.newKey)
 		}
 	}
+	if txn.IsPipelined() {
+		return txn.Prefetch(ctx, batchKeys)
+	}
 	return txn.BatchGet(ctx, batchKeys)
```

Comment:
Is it necessary to do this `txn.BatchGet` for a pipelined txn?

Reply:
Union the `Prefetch` with `txn.BatchGet` now.